pipeline/07-export.R

```diff
@@ -285,6 +283,55 @@ assessment_pin_merged <- assessment_pin %>%
   filter(!is.na(pred_pin_final_fmv_land)) %>%
   mutate(across(ends_with("_date"), as_date))
 
+if (comp_enable) {
+  message("Pulling comp data from Athena")
+  # We don't know how many comp_score_{x} columns there are, and selecting
+  # columns via wildcard in SQL is complicated, so just query everything and
+  # then drop metadata columns in R
+  comps <- dbGetQuery(
+    conn = AWS_ATHENA_CONN_NOCTUA, glue("
+    SELECT *
+    FROM model.comp
+    WHERE run_id = '{params$export$run_id}'
+    ")
+  ) %>%
+    select(-run_id, -year, -card)
+
+  # Merge comp data into assessment data
+  assessment_pin_merged <- assessment_pin_merged %>%
+    left_join(comps, by = join_by(meta_pin == pin)) %>%
```

Comment:
**issue:** Comps are card-level while this is PIN-level. Are you sure this isn't making duplicate rows?

Reply:
We talked out of band and decided to just take the best-performing comps across all cards. This is done in e02cdd29b4cc03e629f9f0abeff53a51fccb632b.